cloudwatch_metrics_common/include/cloudwatch_metrics_common/utils/metric_object.h

```diff
@@ -71,21 +71,19 @@ struct MetricObject {
 
   MetricObject(
```

Comment:
nit: docstring to detail this constructor's purpose (stat values)

Reply:
Done.